### PR TITLE
Expand Taiga node

### PR DESCRIPTION
**Pull request by ivov**
This PR expands the Taiga node:

```
epic
  ├── create
  ├── delete
  ├── update
  ├── get
  └── getAll
issue
  ├── create
  ├── delete
  ├── update
  ├── get
  └── getAll
task
  ├── create
  ├── delete
  ├── update
  ├── get
  └── getAll
userStory
  ├── create
  ├── delete
  ├── update
  ├── get
  └── getAll
```

Relates to #1896.

### CODE REVIEW

File: packages/nodes-base/nodes/Taiga/descriptions/UserStoryDescription.ts [hunk 1]
```diff
@@ -0,0 +1,605 @@
...
+			{
+				displayName: 'Status',
+				name: 'status',
+				description: 'ID of the status of the user story',
+				type: 'options',
+				typeOptions: {
+					loadOptionsDependsOn: [
+						'projectId',
+					],
+					loadOptionsMethod: 'getProjectStatuses',
```
Comment (MedAliMarz) [inline packages/nodes-base/nodes/Taiga/descriptions/UserStoryDescription.ts:417]: `getProjectStatuses` is not defined but you meant `getUserStoryStatuses`

**Review by MedAliMarz — COMMENTED** [review]
@ivov  Thank you for extending the node, the added operation works correctly. Maybe moving the projectID from the optional field up to the required field will help in avoiding buggy behavior when loading options that rely on the projectID value.

**Comment by cmprmsd** [comment]
@ivov Cool! This looks so much cleaner than my "make it work" attempt xD 
Hope yours get's merged 👍👌

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with n8n@0.130.0

**Comment by cmprmsd** [comment]
Nice! Will test it asap 🥳

**Pull request merged**